products/sle/main.pm

```diff
@@ -76,9 +76,10 @@ sub default_desktop {
     return 'gnome' if get_var('VERSION', '') lt '15';
     # with SLE 15 LeanOS only the default is textmode
     return 'gnome' if get_var('BASE_VERSION', '') =~ /^12/;
-    # In sle15 we add repos manually to make a workaround of missing SCC, gnome will be installed as default system.
-    return 'gnome' if get_var('ADDONURL', '') =~ /(desktop|server)/;
     return 'textmode' if (get_var('SYSTEM_ROLE') && !check_var('SYSTEM_ROLE', 'default'));
+    # default system role for sles and sled
+    return 'textmode' if sle_version_at_least('15') && is_server;
```

Comment:
as far as I know SYSTEM_ROLE is defined only for SLE15 , right ? 
so line 79 can be removed also

Reply:
also using sle_version_at_least is confusing for me because line 76 already prove us that version is not smaller '15' string

Reply:
We have xen tests, so I would be carefull with SYSTEM_ROLES, but good catch regarding sle_version_at_least